x/evm/client/cli/tx_proposal.go

```diff
@@ -370,3 +371,77 @@ func CmdEvmProposalSetFeeManagerAddress() *cobra.Command {
 
 	return cmd
 }
+
+func CmdEvmProposalSetSmartContractDeployer() *cobra.Command {
```

Comment:
I think we should use away from legacy proposals like this. Using the established proposal flow using draft files is easier to use, and we don't have to write a new CLI endpoint for every proposal.